src/features/unified-share-modal/ContactsField.js

```diff
@@ -209,6 +209,7 @@ class ContactsField extends React.Component<Props, State> {
                 placeholder={intl.formatMessage(commonMessages.pillSelectorPlaceholder)}
                 ref={fieldRef}
                 selectedOptions={selectedContacts}
+                showAvatars
```

Comment:
added so current behavior does not change since `showAvatar` was previously always set for `RoundPill` inside of `PillSelector`

Reply:
Should we default `showAvatars` to true?

Reply:
I thought about that so it doesn't break current behavior but because the usages are so low, I was going to manually fix them in a bump.

one reason I was against it was because `RoundPill` (the component that does the avatar stuff) defaults to `false`. so having the a parent component default to `true` seemed odd: https://github.com/box/box-ui-elements/blob/master/src/components/pill-selector-dropdown/RoundPill.js#L41

the other reason:
```
// if we default to true, then components will need to do this
<PillSelectorDropdown
  showAvatars={false}
  showRoundPills
/>

----------------

// defaulting to undefined seems ideal because we can do:

// won't show avatars
<PillSelectorDropdown
  showRoundPills
/>

// will show avatars
<PillSelectorDropdown
  showAvatars
  showRoundPills
/>

// showAvatars and showRoundPills have the same usage which is good because they're used together related
```

Reply:
All good points. Found this related post about boolean variable conventions: https://www.serendipidata.com/posts/naming-guidelines-for-boolean-variables

Maybe if we wanted to default to not showing avatars, we should have a prop `hideAvatars` instead that defaults to false. But since we already have `showAvatar` I'm good with sticking to `showAvatars` and defaulting to false.